python/cuml/common/__init__.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
Perhaps we should add some imports here and in the `__init__.py` in the base `python/cuml` repo to make importing base less verbose in the inheriting classes (i.e. `from cuml import Base` instead of `from cuml.common.base import Base`). See the imports included in the other folders and in base for example.

This could be true of some of the other classes that we want to import constantly